_docs_integrate/request-one-time-consent-of-peer.md

```diff
@@ -25,88 +25,107 @@ required_by:
+- "I hereby confirm the login to the website."
+- "I confirm my attendance at the course."
 
-```json
-{
-  "content": {
-    "items": [
-      {
-        "@type": "ConsentRequestItem",
-        "mustBeAccepted": true,
-        "title": "The Sender is asking for an one time consent",
-        "consent": "I hereby confirm that I have read the privacy terms of this cloud service and agree to them.",
-        "link": "privacy.demo"
-      }
-    ]
-  },
-  "peer": "<the address of the Recipient Connector>"
-}
-```
+The `consent` property of a [ConsentRequestItem]({% link _docs_integrate/data-model-overview.md %}#consentrequestitem) is not intended to be used by an Identity to send tons of text to the peer.
+Instead, it should contain a brief summary of the issue, which the peer should agree to.
+Longer texts should be placed on external websites.
+A link to such a website can be specified in the optional `link` property of the ConsentRequestItem.
+Also note that the ConsentRequestItem should not be used for contractual agreements.
 
-### Examples {#consentrequestitem-examples}
+## Request for one-time consent
 
-- "I hereby confirm the login to the website"
-- "I confirm my presence in the course"
+In the following, we describe how a Connector, hereinafter referred to as the Sender, can get the one-time consent of another Connector, with which it has already established a Relationship and which is also referred to as the Recipient, by using a [ConsentRequestItem]({% link _docs_integrate/data-model-overview.md %}#consentrequestitem) within the `items` property of an appropriate [Request]({% link _docs_integrate/data-model-overview.md %}#request).
 
-Please do not use the ConsentRequest to submit tons of text to the peer Identity. It is meant to be a short consent or summary the user agrees to. Please move longer text to external websites.
-The ConsentRequest is also not meant for contractual agreements.
+The general procedure is the same if the Connector wants to obtain the one-time consent of an App user instead of another Connector. For reasons of clarity, this guide focuses on the process with two Connectors. For information on how to establish Relationships, refer to the [Establish Relationships]({% link _docs_integrate/establish-relationships.md %}) scenario documentation.
 {: .notice--info}
 
-## Create the Request
+As there is already a Relationship between the Sender and the Recipient, the Sender can send the [Request via a Message]({% link _docs_integrate/requests-via-messages.md %}) to the Recipient. Even if the Relationship has already been established, the Sender could also send the [Request via a RelationshipTemplate]({% link _docs_integrate/requests-via-relationshiptemplates.md %}) to the Recipient, but this is not discussed further below.
```

Comment:
>  but this is not discussed further below

It sounds as if the "further" refers to "below", although I assume you want it to refer to "discussed". An easy clarification could be yielded replacing "below" with "here".